unyt/tests/test_unyt_array.py

```diff
@@ -498,6 +498,12 @@ def test_power():
     with pytest.raises(UnitOperationError):
         np.power(cm_arr, unyt_quantity(3, "g"))
 
+    try:
+        np.power(cm_arr, unyt_quantity(3, "g"))
+    except UnitOperationError as err:
+        assert isinstance(err.unit1, Unit)
+        assert isinstance(err.unit2, Unit)
+
```

Comment:
Good catch on the bug!

Assuming my reading of the pytest docs is correct, I'm pretty sure this can be phrased like this:

```
with pytest.raises(UnitOperationError) as excinfo:
    assert type(excinfo.value.unit1) is Unit
    assert type(excinfo.value.unit2) is Unit
```

However I think the way you have it is actually simpler so I'm going to merge it as-is.